### PR TITLE
Minor misalignment between RDF and images solved

### PR DESCRIPTION
**Pull request by pabloalarconm**

### CODE REVIEW

File: example-data/turtle/geneticDiagnosis.ttl [hunk 1]
```diff
@@ -41,7 +41,7 @@
     sio:SIO_000300 "https://www.omim.org/entry/192977"^^xsd:string ;
     sio:SIO_000628 <https://www.omim.org/entry/192977> .
 
-<https://www.omim.org/entry/192977> a sio:SIO_000614, sio:SIO_000015, obo:NCIT_C173071 ;
+<https://www.omim.org/entry/192977> a sio:SIO_000614, sio:SIO_000015, obo:NCIT_C171178 ;
```
Comment (rajaram5) [inline example-data/turtle/geneticDiagnosis.ttl:44]: This node is of type gene right? Why `obo:NCIT_C171178 (Sequence Variant Report)` used as a type here?

File: example-data/turtle/geneticDiagnosis.ttl [hunk 2]
```diff
@@ -53,5 +53,5 @@
     sio:SIO_000300 "https://identifiers.org/HGNC:12698"^^xsd:string ;
     sio:SIO_000628 <https://identifiers.org/HGNC:12698> .
 
-<https://identifiers.org/HGNC:12698> a sio:SIO_000614, sio:SIO_000015, obo:NCIT_C173071 ;
+<https://identifiers.org/HGNC:12698> a sio:SIO_000614, sio:SIO_000015, obo:NCIT_C171178 ;
```
Comment (rajaram5) [inline example-data/turtle/geneticDiagnosis.ttl:56]: This node is of type gene right? Why `obo:NCIT_C171178 (Sequence Variant Report)` used as a type here?

**Review by rajaram5 — CHANGES_REQUESTED** [review]
There some minor issues can you address them

File: example-data/turtle/diagnosis.ttl [hunk 3]
```diff
@@ -24,11 +24,11 @@
...
 :diagnosis_output_ a sio:SIO_000015, sio:SIO_001003 ;
     rdfs:label "Output Type: Diagnosis"^^xsd:string ;
-    sio:SIO_000300 "rare non-syndromic central nervous system malformation"^^xsd:string ;
+    sio:SIO_000300 "extremely rare neurodegenerative disorder characterized by progressive spinocerebellar ataxia"^^xsd:string ;
     sio:SIO_000628 :diagnosis_attribute_ .
 
-:diagnosis_attribute_ a sio:SIO_000614, <http://www.orpha.net/ORDO/Orphanet_1400> ;
-    rdfs:label "rare non-syndromic central nervous system malformation"^^xsd:string .
+:diagnosis_attribute_ a sio:SIO_000614, <http://www.orpha.net/ORDO/Orphanet_1399> ;
+    rdfs:label "Attribute Type: extremely rare neurodegenerative disorder characterized by progressive spinocerebellar ataxia"^^xsd:string .
```
[the remark this answers is not in the document]
Comment (pabloalarconm) [reply example-data/turtle/diagnosis.ttl:31]: Its a richer textual description from `Richards-Rundle syndrome` OLS webpage. so yes, it fits with the requirement from docs:

diagnostic_opinion: brief textual label of the diagnosic opinion. Likely the label of the ORDO disease term, but may be richer.

File: example-data/turtle/geneticDiagnosis.ttl (hunk 2 above)
Comment (pabloalarconm) [reply example-data/turtle/geneticDiagnosis.ttl:56]: Based on current YARRRML templates, all three (HGVS, OMIM and HGNC) have `obo:NCIT_C171178 (Sequence Variant Report)` as type

File: example-data/turtle/geneticDiagnosis.ttl (hunk 1 above)
Comment (pabloalarconm) [reply example-data/turtle/geneticDiagnosis.ttl:44]: Based on current YARRRML templates, all three (HGVS, OMIM and HGNC) have `obo:NCIT_C171178 (Sequence Variant Report)` as type

File: example-data/turtle/diagnosis.ttl (hunk 3 above)
[the remark this answers is not in the document]
Comment (rajaram5) [reply example-data/turtle/diagnosis.ttl:31]: so will you change the label of this node?

**Pull request merged**